code-of-conduct/about-code-of-conduct-incident-record.md

```diff
@@ -0,0 +1,93 @@
+# Code of Conduct incident record template guide
+
+---
+Thank you for downloading the Code of Conduct template from The Good Docs Project!
+We welcome any kind of feedback on our templates, both positive and negative.
+To suggest improvements, open an issue on our (https://github.com/thegooddocsproject/templates/issues)[Templates repository].
+
+This is the guide that explains how to use the Good Docs Project Code of Conduct incident record, which is part of the Code of Conduct template set.
+
+The Code of Conduct template set includes:
+
+* A Code of Conduct that you can use to create and explain your community's Code of Conduct. This template is included in the same folder as this template and is named `template-code-of-conduct.md`. See its accompany guide: `about-code-of-conduct.md`.
+* A Code of Conduct response plan template that you can use to create and explain the policy your team will follow as you handle Code of Conduct incidents. This template is included in the same folder as the Code of Conduct template and is named `template-code-of-conduct-response-plan.md`. See its accompany guide: `about-code-of-conduct-response-plan.md`.
+* A Code of Conduct incident record template for a form that is filled out when a community moderator takes an incident report from a community member. This template is included in the same folder as the Code of Conduct template as the Code of Conduct template and is named `template-code-of-conduct-incident-record.md`. See its accompany guide: `about-code-of-conduct-incident-record.md`.

```

Comment:
suggestion (polish): "This template is included in the same folder as the Code of Conduct template ~~as the Code of Conduct template~~..."

Reply:
I see that this chunk of text in reused in the other template guides, so I won't add more comments